dynamic_rest/viewsets.py

```diff
@@ -448,7 +491,7 @@ def _create_many(self, data):
                 serializer.is_valid(raise_exception=True)
             except exceptions.ValidationError as e:
                 errors.append({
-                    'detail': str(e),
+                    'detail': e.detail,
```

Comment:
Drive-by: `e.detail` is often a dictionary, `str(e)` returns a nasty string-encoded version of the dictionary